.travis.yml

```diff
@@ -3,3 +3,4 @@ node_js:
   - "6"
 notifications:
   email: false
+script: npm run test && node check_version.js
```

Comment:
good idea adding this into CI, it seems like something that would be easy to forget to update